gulpfile.js

```diff
@@ -72,18 +59,44 @@ function getPlatforms(includeChromeOs) {
     return platforms;
 }
 
-function getRunDebugAppCommand() {
+// Gets the default platform to be used
+function getDefaultPlatform() {
+    var defaultPlatform;
     switch (os.platform()) {
     case 'darwin':
-        return 'open ' + path.join(debugDir, pkg.name, 'osx64', pkg.name + '.app');
+        defaultPlatform = 'osx64';
 
         break;
     case 'linux':
-        return path.join(debugDir, pkg.name, 'linux64', pkg.name);
+        defaultPlatform = 'linux64';
+
+        break;
+    case 'win32':
+        defaultPlatform = 'win32';
+
+        break;
+        
+    default:
+        defaultPlatform = os.platform();
```

Comment:
I suspect that this will break for most cases. Do you have any examples where this actually works? If not, this should just return null.

Reply:
This was not a problem, the objective of this function was to "return" the "most similar" name of the platform that must be executed by default. If not found, it returns the actual platform but this function is only called from the `getPlatforms()` method, and the code, if the returned platform is not one of the "compatibles" it does not add it to the list of platforms to be executed.
I've changed it, no problem, returning empty string and not null (the rest of the code returns an empty string when something is not found or not valid). Do you prefer that I use a null?